### PR TITLE
Add translation to PT-BR

### PR DESCRIPTION
**Pull request by weslleymurdock**
Translation of Iterables and Generators.md file

As off #233, tagging @khaosdoctor, @alvarocamillont, @danilofuchs e @orta

### CODE REVIEW

**Comment by khaosdoctor** [comment]
I'll review it right now. @weslleyluiz can you just edit the PR title to "Add <filename> translation to PT-BR"?

**Comment by weslleymurdock** [comment]
Sorry, my bad. I've changed the title, thanks for the observation

File: packages/documentation/copy/pt/reference/Iterators and Generators.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+let someArray = [1, "string", false];
+
+for (let entry of someArray) {
+  console.log(entry); // 1, "string", false
+}
+```
+
+### Declarações `for..of` vs. `for..in`
+
+Ambas declarações `for..of` e `for..in` iteram em listas; os valores iterados são diferentes, porém, `for..in` retorna uma lista de _keys_ no objeto que está sendo iterado, enquanto `for..of` retorna uma lista de _values_ das propriedades numéricas do objeto que está sendo iterado.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/reference/Iterators and Generators.md]:
```suggestion
Ambas declarações `for..of` e `for..in` iteram em listas; os valores iterados são diferentes, porém, `for..in` retorna uma lista de _chaves__ no objeto que está sendo iterado, enquanto `for..of` retorna uma lista de _valores_ das propriedades numéricas do objeto que está sendo iterado.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Just a small part that should be translated

File: packages/documentation/copy/pt/reference/Iterators and Generators.md (hunk 1 above)
Comment (weslleymurdock) [reply packages/documentation/copy/pt/reference/Iterators and Generators.md]: Requested changes made

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**